stan/math/fwd/fun/mdivide_left.hpp

```diff
@@ -19,6 +19,9 @@ inline Eigen::Matrix<fvar<T>, R1, C2> mdivide_left(
     const Eigen::Matrix<fvar<T>, R2, C2> &b) {
   check_square("mdivide_left", "A", A);
   check_multiplicable("mdivide_left", "A", A, "b", b);
+  if (A.size() == 0) {
+    return Eigen::Matrix<fvar<T>, R1, C2>(0, b.cols());
```

Comment:
This should just be `return {0, b.cols()};` The constructor is redundant given that it's the return type.

I'm only going to marke this one, but this is a problem with almost every return defined in this PR.

Reply:
I'm glad this works! When I tried something similar in a different PR, it wouldn't compile. Perhaps because here the return type is spelled out unambiguously.